ext/pdo_sqlite/sqlite_statement.c

```diff
@@ -375,13 +375,22 @@ static int pdo_sqlite_stmt_get_attribute(pdo_stmt_t *stmt, zend_long attr, zval
 		case PDO_SQLITE_ATTR_READONLY_STATEMENT:
 			ZVAL_FALSE(val);
 
-#if SQLITE_VERSION_NUMBER >= 3007004
+#if SQLITE_VERSION_NUMBER >= 3007004 // Should we still keep it ? avail. since circa 2010
```

Comment:
Depends on the version requirement in the autoconf file

Reply:
How did you choose 3.25? I bet some ancient distros have some even older versions

Reply:
as mentioned in the commit message it s open for discussion, I would not recommend lower than 3.7.17 which even centos 7 support while we plan to increase minimum centos release supported (8).

Reply:
same as centos 7, probably some alternative repositories offered newer versions but that s the official one as far as I m concerned.